ideascube/monitoring/forms.py

```diff
@@ -73,16 +74,17 @@ class Meta:
 
 class InventorySpecimenForm(forms.ModelForm):
 
-    specimen = forms.CharField(widget=forms.TextInput(attrs={
-                               'placeholder': _('Enter a bar code')}))
+    specimen = forms.CharField(label=_('specimen'),
```

Comment:
I hate hate hate how Django uses the same term "field" for both models and forms. So much confusion! >_<